src/Common/src/Interop/Windows/BCrypt/Interop.NTSTATUS.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal partial class Interop
+{
+    internal partial class BCrypt
+    {
+        internal enum NTSTATUS : uint
```

Comment:
Nit: NTSTATUS is actually signed 32 int in Windows SDK. It maybe better just keep these error constants in Interop.BCryptGenRandom.cs as they used to be; not worth it to create enum for it.